src/test/java/fm/last/moji/integration/AbstractMojiIT.java

```diff
@@ -133,4 +133,62 @@ String newKey() {
     return newKey(RandomStringUtils.randomAlphanumeric(16));
   }
 
+  private void initMogileTestData() throws IOException {
+    if(initMogileTestDataIsDone) {
```

Comment:
I think if it only adds a few seconds that's fine, this is a huge improvement over the time required before to get and keep everything else up and running I'd say go with @BeforeClass for now and later if we really have an issue with slow tests we can come back to this.